redaxo/src/core/lib/login/user.php

```diff
@@ -226,4 +248,22 @@ public static function setRoleClass($class)
     {
         self::$roleClass = $class;
     }
+
+    /**
+     * Removes the instance of the given key.
+     *
+     * @param mixed $key Key
+     */
+    public static function clearInstance($key)
+    {
+        $key = self::getInstancePoolKey($key);
+        $class = static::class;
+
+        # clear login cache for instance
+        $instance = self::$instances[$class][$key];
+        $login = $instance->getLogin();
```

Comment:
Hier kommt es wahrscheinlich zu einem Fehler, wenn es gar keine Instance gab zu der ID, die übergeben wurde, oder?

Reply:
Habe nicht an dieses Szenario gedacht und habe es nicht getestet, aber ich vermute du hast Recht. Ich habe nun einen Check eingebaut.

Ich werde im Laufe der Woche wieder eine Redaxo Instanz aufsetzten und denn PR noch testen.

Reply:
Der Fehler wird schon in der hier markierten Zeile passieren. Also `$instance` kann `null` sein, 
und `$instance->getLogin()` schlägt dann fehl.

Reply:
Ich habe es nun gelöst.